scania-digital-design-system/sdds
# fix/remove inline svgs footer/header/side menu

**Describe pull-request**  
Remove all hard-coded svg from footer/side-menu/header and replaced it with sdds-icon or webfont solution.

**Solving issue**  
Fixes: [AB#2705](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2705)

**How to test**  
1. Go to storybook link below
2. Check in Components -> Header/Footer/Side-menu
3. Check that all stories still look good. 

**Screenshots**  
-

**Additional context**  
-
